src-files/def-config.yaml

```
@@ -48,6 +48,9 @@ verification:
   # wait this amount of time before attempting to verify the link again.
   defaultRetryAfter: 30s
 
+  # Count of attempts of getting access to remote data (links to the sites, etc)
```

Let's be a bit more specific here:

```suggestion
  # How many attempts to retry an external link after getting
  # a "429 Too Many Requests" response.
```

---

Please update the CLI option's help text as well.